exercise/exercise_models.py

```diff
@@ -566,7 +566,7 @@ def _check_submission_allowed(self, profile, request=None):
         # Support group id from post or currently selected group.
         group = None
         group_id = request.POST.get("_aplus_group") if request else None
-        if not group_id is None:
```

Comment:
`request.POST.get("_aplus_group")`
Did you check if the code finds any group in this case? Normally, students create a group in a separate page first and then they can choose the group when they are making the submission to the exercise. However, when the teacher modifies the submitters of a submission in the "inspect submission" page, the submission has a group even though nobody has created the group in the database. Is the `_aplus_group` hidden field empty then?